content/en/docs/tasks/debug-application-cluster/debug-service.md

```diff
@@ -91,49 +92,84 @@ metadata:
 spec:
   selector:
     matchLabels:
-      app: hostnames
+      run: hostnames
   replicas: 3
   template:
     metadata:
       labels:
-        app: hostnames
+        run: hostnames
     spec:
       containers:
       - name: hostnames
         image: k8s.gcr.io/serve_hostname
-        ports:
-        - containerPort: 9376
-          protocol: TCP
 ```
+
+The label "run" is automatically set by `kubectl run` to the name of the
+`Deployment`.
 {{< /note >}}
 
-Confirm your `Pods` are running:
+You can confirm your `Pods` are running:
 
 ```shell
-kubectl get pods -l app=hostnames
+kubectl get pods -l run=hostnames
 NAME                        READY     STATUS    RESTARTS   AGE
 hostnames-632524106-bbpiw   1/1       Running   0          2m
 hostnames-632524106-ly40y   1/1       Running   0          2m
 hostnames-632524106-tlaok   1/1       Running   0          2m
```

Comment:
The style guide recommends [separating commands from output](https://kubernetes.io/docs/contribute/style/style-guide/#separate-commands-from-output). If you want to, convert blocks like this to follow those recommendations.

Reply:
Can do